src/index.html

```diff
@@ -2,16 +2,190 @@
+            maxlength="100"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="field__wrapper">
+          <label for="name">Name: </label>
+
+          <input
+            type="text"
+            name="name"
+            id="name"
+            minlength="3"
+            maxlength="100"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="field__wrapper">
+          <label for="age">How old are You? </label>
+
+          <input
+            type="number"
+            name="age"
+            id="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </div>
+
+        <div class="field__wrapper">
+          <label for="date">Full date of birth: </label>
+
+          <input
+            type="date"
+            name="date"
+            id="date"
+          >
+        </div>
+
+        <label for="check">I accept the term of the agreement </label>
+
+        <input
+          type="checkbox"
+          value="check"
+          name="check"
+          id="check"
+        >
+      </fieldset>
+
+      <fieldset class="box__style">
+        <legend>Registration</legend>
+
+        <div class="field__wrapper">
+          <label for="email">E-mail: </label>
+
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            multiple
+          >
+        </div>
+
+        <label for="user-password">Password: </label>
+
+        <input
+          name="user-password"
+          id="user-password"
+          type="password"
+          required autocomplete="off"
+        >
+      </fieldset>
+
+      <fieldset class="box__style">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="field__wrapper">
+          <label for="cat">Do you love cats? </label>
+
+          <input
+            type="radio"
+            id="yes"
+            name="cat"
+            value="Y"
+          >
+          <label for="yes">Yes</label>
+
+          <input
+            type="radio"
+            id="no"
+            name="cat"
+            value="N"
+          >
+          <label for="no">No</label>
+        </div>
+
+        <div class="field__wrapper">
+          <label for="color">What is your favorite color? </label>
+
+          <input
+            type="color"
+            name="color"
+          >
+        </div>
+
+        <div class="field__wrapper">
+          <label for="time">What time do you go to bed? </label>
+
+          <input
+            type="time"
+            name="time"
+            id="time"
+            step="2"
+          >
+        </div>
+
+        <div class="field__wrapper">
+          <label for="cars">What are your favorite brand of cars? </label>
+
+          <textarea
+            name="cars"
+            id="cars"
+            rows="4"
+            cols="2"
+          >BMW Audi Lada
+          </textarea>
```

Comment:
Here more suitable would be to use select tag instead of textarea 
Here is an example:
```html
<select name="cars" id="cars" multiple>
  <option value="volvo">Volvo</option>
  <option value="saab">Saab</option>
  <option value="opel">Opel</option>
  <option value="audi">Audi</option>
</select>
```

This will also fix your test results :)